packages/bsky/src/db/migrations/20230927T202833377Z-create-moderation-subject-status.ts

```diff
@@ -0,0 +1,55 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .alterTable('moderation_action')
+    .renameColumn('reason', 'comment')
+    .execute()
+  await db.schema
+    .alterTable('moderation_action')
+    .alterColumn('comment')
+    .dropNotNull()
+    .execute()
+  await db.schema
+    .alterTable('moderation_action')
+    .addColumn('refEventId', 'integer')
+    .execute()
+  await db.schema
+    .alterTable('moderation_action')
+    .addColumn('meta', 'jsonb')
+    .execute()
+  await db.schema
+    .createTable('moderation_subject_status')
+    .addColumn('id', 'serial', (col) => col.primaryKey())
+    .addColumn('subjectType', 'varchar', (col) => col.notNull())
+    .addColumn('subjectDid', 'varchar', (col) => col.notNull())
+    .addColumn('subjectUri', 'varchar')
+    .addColumn('subjectCid', 'varchar')
+    .addColumn('createdAt', 'varchar', (col) => col.notNull())
+    .addColumn('updatedAt', 'varchar', (col) => col.notNull())
```

Comment:
will probably want an index on `updatedAt` also (to have a view/query of "recent mod actions", and possibly for pagination), but could wait until we have an actual need to create it